CHANGELOG.md

```diff
@@ -5,7 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## Unreleased
+## [Unreleased]
```

Comment:
Gotta add the link below if you want this to be a link like:

[Unreleased]: https://github.com/Esri/calcite-components/compare/v1.0.0-beta.32...master "Unreleased"